pages/Javascript-API.md

```diff
@@ -60,25 +62,86 @@ The promise resolves with a `selection` object:
 - `pixel`: the XY location within the map container where the event occurred, in the form `{ x, y }`
 - `leaflet_event`: the Leaflet event that triggered the selection
 
-#### `load(scene_url, base_path)`
-Loads the specified scene by url and rebuilds the geometry. If no arguments are specified, the current scene will be reloaded.
+In addition to the `scene.getFeatureAt()` method, the `radius` parameter may be set when creating or configuring the Leaflet layer:
+
+- Leaflet layer instantiation:
+```yaml
+layer = Tangram.leafletLayer({
+   scene: 'path/to/scene.yaml',
+   events: {
+      click: clickHandler,
+   },
+   selectionRadius: 10 // radius of 10px
+};
+```
+
+- Leaflet layer interface for setting selection handlers:
+```yaml
+layer.setSelectionEvents (events, { radius: 5 }) // radius of 5px
+```
+
+When using a radius, the feature closest to the center point will be returned. As with existing feature selection, only features marked as interactive: true will register.
+
+#### `load(scene_url, { base_path, file_type })`
+Loads the specified scene by URL and rebuilds the geometry. If no arguments are specified, the current scene will be reloaded.
 
-`scene_url` is the path to a scene file. By default, relative paths within this file (for images and other resources) are relative to its host url.
+`scene_url` is the path to a scene file. By default, relative paths within this file (for images, fonts, or other resources) are relative to this URL. For example, for a scene loaded from `https://example.com/path/to/scene.yaml`, an image `texture.png` referenced in that scene file would resolve to `https://example.com/path/to/texture.png`.
+
+Other options may be passed inside an object.
 
 `base_path` is an optional argument specifying an alternate base URL for resolving relative paths in the scene file. It is primarily useful for development and debugging.
 
+```javascript
+scene.load(scene_url, { base_path: 'https://site.com/scene/resources/' });
+```
+
+`file_type` is an optional argument to support cases where zipped files may be loaded locally, such as from a blob URL.
+
+Using `Scene.load()`:
+
+```javascript
+scene.load(zip_blob_url, { file_type: 'zip' });
+```
+
+Using `Tangram.leafletLayer`:
+
+```javascript
+layer = Tangram.leafletLayer({
+   scene: zip_blob_url,
+   sceneFileType: 'zip',
+   ...
+});
+```
+
 #### `loadTextures()`
 Reloads and rebinds [textures](textures.md) in the scene.
 
 Textures will only be reloaded if any of the texture definition's parameters changed, or if the texture is tied to a DOM element.
 
+#### `queryFeatures()`
```

Comment:
There are several other options we need to incorporate here, such as `visible`, `unique`, `group_by`, `geometry`. These should be well documented (along with some examples that I think are worth incorporating) in https://github.com/tangrams/tangram/pull/566.